Gems/GraphCanvas/Code/CMakeLists.txt

```diff
@@ -78,3 +80,35 @@ if (PAL_TRAIT_BUILD_HOST_TOOLS)
 
 
 endif ()
+
+################################################################################
+# Tests
+################################################################################
+if(PAL_TRAIT_BUILD_TESTS_SUPPORTED)
+
+    ly_add_target(
+        NAME GraphCanvas.Tests ${PAL_TRAIT_TEST_TARGET_TYPE}
+        NAMESPACE Gem
+        FILES_CMAKE
+            graphcanvas_tests_files.cmake
+        COMPILE_DEFINITIONS
+            PRIVATE
+                GRAPHCANVAS_TESTS
```

Comment:
is this required? don't see it get used

Reply:
Seemed fairly standard among the examples I saw, I could remove it